homeassistant/components/vizio/media_player.py

```diff
@@ -196,6 +201,11 @@ def supported_features(self):
         """Flag device features that are supported."""
         return self._supported_commands
 
+    @property
+    def _unique_id(self):
```

Comment:
This property should be `unique_id`, not `_unique_id`.

Reply:
Thanks, resolved